library_generation/test/resources/proto_path_list.txt

```diff
@@ -5,13 +5,13 @@
 # google/bigtable/v2 java-bigtable true
 google/cloud/apigeeconnect/v1 java-apigee-connect false
 google/cloud/asset/v1p5beta1 java-asset false
-google/cloud/asset/v1p2beta1 java-asset false
+# google/cloud/asset/v1p2beta1 java-asset false
```

Comment:
Why do we disable this test?

Reply:
I forgot to add a mention of https://github.com/googleapis/sdk-platform-java/issues/2303 - I will add it to the description

Some googleapis protos get updated before they appear in googleapis-gen or google-cloud-java